### ISSUE TITLE
Question about implementation

### ISSUE THREAD

**Issue by Florianisme** [FE-H/XAdES-T-Example#1]
Hey, did you get this to work? We are currently implementing the same interface to the Malaysian authority and do not get the Signature part to work

**Comment by FE-H** [comment]
Hi, I stopped the exploration mode for quite sometime as we have onboard a dedicated Java resource. Our resource made a custom solution instead of using the library.

Based on what I've gathered, ESIG DSS will also output other XaDES-T elements not specified in the IRB SDK. The only way to influence this is to customise the library - did not reach out to the original authors of this library for clarification at the time. I also could not get the verification part working after signing or to disable it.

You might want to explore if XADES4J library could work. To my memory, both of these libraries also output certain elements that are not listed as a part of the IRB SDK.

**Comment by Florianisme** [comment]
Hi, thanks for the response!
I find the IRB SDK description a bit conflicting regarding how they expect the document to look like before/after it's signed.

If you would be okay with it I hoped we could exchange further information via email.

**Comment by FE-H** [comment]
Yeah, sure would be happy to discuss over the email. Not sure where I can share it though - don't see a private/direct message on my mobile app

**Comment by Florianisme** [comment]
Contact me at florian.m9782@gmail.com :)
